.github/workflows/build.yml

```diff
@@ -274,6 +274,33 @@ jobs:
             ${{ github.workspace }}/packages/**/*.tgz
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
+  job_playwright_test:
+    name: Playwright E2E Tests
+    needs: job_build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out current commit (${{ github.sha }})
+        uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v1
```

Comment:
Why not `setup-node@v2`?

Reply:
Just as the other jobs are currently using `setup-node@v1`. I can try to upgrade all at once, though?

Reply:
Oops, didn't notice. We can keep `v1`, bumping the version should be done in another PR and we must be sure it will all work as expected.